### PR TITLE
net-p2p/deluge: ebuild improvements

### PR DESCRIPTION
**Pull request by PPed72**
Split startup scripts for daemon and web interface. Fixes Gentoo bug #350213.

### CODE REVIEW

**Comment by monsieurp** [comment]
@krifisk @gentoo/proxy-maint

File: net-p2p/deluge/files/deluge-web.init [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/sbin/runscript
```
Comment (anthonyryan1) [inline net-p2p/deluge/files/deluge-web.init]: `/sbin/runscript` has been deprecated in favour of `/sbin/openrc-run`

File: net-p2p/deluge/files/deluged.init-2 [hunk 2]
```diff
@@ -0,0 +1,54 @@
+#!/sbin/runscript
```
Comment (anthonyryan1) [inline net-p2p/deluge/files/deluged.init-2]: `/sbin/runscript` has been deprecated in favour of `/sbin/openrc-run`

**Comment by krifisk** [comment]
Thank you for this PR, I'll review it and get back to you (probably this weekend)

**Comment by krifisk** [comment]
Thank you. PR pushed, feel free to close the relevant bugs

**Pull request closed without merging**